modules/serverless-install-serving-web-console.adoc

```diff
@@ -14,23 +14,22 @@ image::serving-installed-operator.png[Installed Operators page]
 . Click the *Create Knative Serving* button.
 +
 image::serverless-create-serving.png[Knative Serving tab]
-. In the *Create Knative Serving* page, you can choose to configure the `KnativeServing` object by using either the default form provided, or by editing the YAML.
-+
-* Using the form is recommended for simpler configurations that do not require full control of `KnativeServing` object creation.
-+
-Optional. If you are configuring the `KnativeServing` object using the form, make any changes that you want to implement for your Knative Serving deployment.
-+
-After you complete the form, click *Create*.
+. In the *Create Knative Serving* page, you can choose to configure the `KnativeServing` object by using either the form provided, or by editing the YAML.
 +
 image::serving-form-view.png[Create Knative Serving in Form View]
 +
-* Editing the YAML is recommended for more complex configurations that require full control of `KnativeServing` object creation. You can access the YAML by clicking the *edit YAML* link in the top right of the *Create Knative Serving* page.
-+
-Optional. If you are configuring the `KnativeServing` object by editing the YAML, make any changes to the YAML that you want to implement for your Knative Serving deployment.
+image::serverless-create-serving-yaml.png[Create Knative Serving in YAML view]
 +
-After you have finished modifying the YAML, click *Create*.
+* Using the form is recommended for simpler configurations that do not require full control of `KnativeServing` object creation.
+* Editing the YAML is recommended for more complex configurations that require full control of `KnativeServing` object creation. You can access the YAML by clicking the *edit YAML* link in the top right of the *Create Knative Serving* page.
 +
-image::serverless-create-serving-yaml.png[Create Knative Serving in YAML view]
+After you complete the form, or have finished modifying the YAML, click *Create*.
+
+[NOTE]
+====
+For more information about configuration options for the KnativeServing custom resource definition, see the documentation on _Advanced installation configuration options_.
+====
+
```

Comment:
Based on the preview-bot preview, the following step starts with 1. Is this expected? If not, maybe a `+` before the NOTE?